Wox.Core/Plugin/PluginsLoader.cs

```diff
@@ -110,5 +110,18 @@ public static IEnumerable<PluginPair> PythonPlugins(IEnumerable<PluginMetadata>
             });
             return plugins;
         }
+
+        public static IEnumerable<PluginPair> ExecutablePlugins(IEnumerable<PluginMetadata> source)
+        {
+            var metadatas = source.Where(o => o.Language.ToUpper() == AllowedLanguage.Executable);
+
+            var plugins = metadatas.Select(metadata => new PluginPair
+            {
+                Plugin = new ExecutablePlugin(metadata.ExecuteFilePath),
```

Comment:
The arguments for constructor is the executable for that language (pythonw etc) , not the executable for plugin (main.lan)